build/build.sh

```diff
@@ -31,8 +31,9 @@ mkdir -p output
 # https://pandoc.org/MANUAL.html
 echo >&2 "Exporting HTML manuscript"
 pandoc --verbose \
-  --defaults="$PANDOC_DEFAULTS_DIR/common.yaml" \
-  --defaults="$PANDOC_DEFAULTS_DIR/html.yaml"
+  --data-dir="$PANDOC_DATA_DIR" \
+  --defaults=common \
```

Comment:
Even though the `.yaml` extension is optional, could leaving it specified make it more obvious to users how they can edit these defaults files?

Reply:
done in https://github.com/manubot/rootstock/pull/340/commits/9b06d3ea990fa1fcb4fb3d19d70b049ec113f6fb